src/BenchmarksApps/Kestrel/PlatformBenchmarks/BenchmarkApplication.cs

```diff
@@ -27,6 +27,8 @@ public partial class BenchmarkApplication
         private readonly static AsciiString _plainTextBody = "Hello, World!";
 
         private static readonly JsonSerializerOptions SerializerOptions = new JsonSerializerOptions();
+        private static readonly JsonMessage _jsonMessage = new JsonMessage { message = "Hello, World!" };
```

Comment:
Doesn't meet the requirements https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#requirements
```
i. For each request, an object mapping the key message to Hello, World! must be instantiated.
```

Reply:
thanks for the catch, I am going to fix it right away